Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -1125,7 +1126,23 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
                   e.printStackTrace();
               }
               break;
-          default: return;
+          case IMAGE_PERMISSION_REQ_CODE_PROFILE:
+              if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
+            // Camera permission is granted, execute the saved callback
+              if (CommonJsInterface.cameraPermissionCallback != null)
+               {
+                CommonJsInterface.cameraPermissionCallback.run();
+                CommonJsInterface.cameraPermissionCallback = null;
+               }
+               } 
+               else 
+                {
+                // Camera permission is denied, handle the situation accordingly (e.g., show a message)
+                    Toast.makeText(activity, "Need Permission To Access The Camere", Toast.LENGTH_SHORT).show();
+                    CommonJsInterface.callingStoreCallImageUpload(juspayServicesGlobal.getDuiCallback(), "", "", "");
```

Comment:
Why the strings are going empty in this callback?

Reply:
because this will give the indication to close the image capture component in the prescript part

Reply:
What if you send the null value here? what will come in purescript side?

Reply:
for null values there is no case in prescript ,
are you suggesting to replace these empty string with null values ?

Reply:
for null values there is no case in prescript ,
are you suggesting to replace these empty string with null values ?

Reply:
Yes If we can